CHANGELOG.md

```diff
@@ -32,6 +32,10 @@
   [lemonspike](https://github.com/LemonSpike)
   [#709](https://github.com/CocoaPods/Xcodeproj/issues/709)
 
+* Do not raise an exception on a Swift Package dependency.
```

Comment:
nit: please add two empty spaces at the end of this line for markdown formattin.

Reply:
done